interfaces/builtin/network_control.go

```diff
@@ -92,6 +92,9 @@ network sna,
 /{,usr/}{,s}bin/wpa_passphrase ixr,
 /{,usr/}{,s}bin/wpa_supplicant ixr,
 
+/dev/rfkill rw,
+/dev/rfkill r,
```

Comment:
I second @zyga here and we should only need `rw` rather than both `rw` and `r`. Which denial do you get when just using `rw`?